x/wasm/internal/keeper/reflect_test.go

```diff
@@ -37,8 +37,9 @@ type reflectPayload struct {
 
 // MaskQueryMsg is used to encode query messages
 type MaskQueryMsg struct {
-	Owner         *struct{} `json:"owner,omitempty"`
-	ReflectCustom *Text     `json:"reflect_custom,omitempty"`
+	Owner       *struct{}               `json:"owner,omitempty"`
+	Capitalized *Text                   `json:"capitalized,omitempty""`
+	Chain       *wasmTypes.QueryRequest `json:"chain,omitempty"`
```

Comment:
These 3 are defined in the reflect class. I added the here thinking we may add more tests for them, since I was translating types from rust to go anyway.